tests/a11y-aria-label-is-well-formatted.js

```diff
@@ -0,0 +1,29 @@
+const rule = require('../lib/rules/a11y-aria-label-is-well-formatted')
+const RuleTester = require('eslint').RuleTester
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+})
+
+const errorMessage =
+  '[aria-label] text should be formatted the same as you would visual text. Use sentence case and make sure you are not using hyphens.'
```

Comment:
What about words with hyphens? Example: `six-pack`/

Reply:
Does this answer your concern: https://github.com/github/eslint-plugin-github/pull/418#issuecomment-1481685661? 

I think it's highly unlike we'll run into a valid, hard-coded `aria-label`. If we do run into one, we can consider adding a way to configure an exception list. Do you think this rule should ship with that?

Reply:
Theoretically, this rule would flag `six-pack` but it wouldn't flag `Six-pack` 😂.

Reply:
Hmmm, what if we also only flag these aria-labels if they don't have a single space?... Technically `six-pack` would still get flagged but it's a more safe check?

Reply:
`SIX-PACK`

Reply:
> what if we also only flag these aria-labels if they don't have a single space?...

I think we'd still want to flag something like:

```
show
hide
close
```

(these are examples I'm pulling from real aria-label we have right now in our codebase)